ocpp/v16/enums.py

```diff
@@ -773,6 +774,7 @@ class UnitOfMeasure(str, Enum):
     k = "K"
     percent = "Percent"
     hertz = "Hertz"
+    b = "Bytes"
```

Comment:
This change is done to the v16 enum. It should be applied to the v201 enum, right?

Reply:
Yes, so it is only an addition now to the doc string